### ISSUE TITLE
Set "resultOf.endTime" in test files to "dummy"

### ISSUE THREAD

**Issue by dr0i** [hbz/lobid-resources#1298]
It's handy to not always have different "resultOf.endTime" with every
deployment. We could set "createEndTime" as morph definition to "0" aka "false"
or "1" aka true to trigger creation of "endTime".

**Comment by dr0i** [comment]
Merged.
From now on it's mandatory to set the morph definition (aka morph variable) `createEndTime`.

**Comment by dr0i** [comment]
Note:
The solution relies on only _one_ morph, triggering the creation of either `dummy` or a timestamp as value for `createEndTime`

I decided against two morphs (one morph just setting `createEndTime` , the other (production one) with an include of the main alma morph and the `createEndTime` morph) because `createEndTime` is part of the `resultOf` entity and that whole entity would have to be part of the `createEndTime` morph, thus the tests would ETL without any `resultOf` metadata (not only lacking the `createEndTime`).

I found the solution (determining from outside of the morph how the morph behaves, independent of the input data) quite good. Not sure if the how-to should be documented - atm it's just noted in the  [https://github.com/metafacture/metafacture-core/wiki/Metamorph-User-Guide](guide) how to use compile-time variables in a static way, as constants .(Of course it's easy to read that constant and trigger behaviour dependent on the value of that constant, but it's not mentioned directly).
